src/System.Text.Utf8/tests/TypeConstraintsTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Reflection;
+using Xunit;
+
+namespace System.Text.Utf8.Tests
+{
+    public class TypeConstraintsTests
+    {
+        [Fact]
+        public void Utf8StringIsAStruct()
+        {
+            var utf8String = "anyString"u8;
```

Comment:
I'm not sure I get this. I think @livarcocc's point is that we are initializing the same value in every test. If we move this to a field then we reduce complexity of each test case. Since each test method does some small variance to the same base scenario setup, having this string defined in each method creates another concept that we need to consider before understanding if the test is broken or if the code is broken.

I'd suggest that we create the field at the class level and initialize it once. Then, if it needs to be modified for a particular instance of a test, it's clear we're diverging on the generalized setup in that case. This will make investigations easier.